docs/notebooks/Kusto-Ingest.ipynb

```diff
@@ -0,0 +1,639 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.                        # Try and parse datetimes, in various formats, to datetimes
</pre><p>Won't pd.to_datetime() do this for you?</p><p>One thing you might have to do is add utc=True param to force TZ-aware</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/msticpy/pull/308/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
That's a better way to achieve the same thing :)